xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/notification/macros.vm

```diff
@@ -193,17 +193,29 @@ $xwiki.getPlainUserName($user)##
 #**
  * Display a notification for a message sent through the Message Stream.
  * TODO: find a way to move it to the "message stream" module
+ *
+ *
+ * @param compositeEvent displays the message of the event
+ * @param translationKey the translation key of the title of the messsage
+ * @param icon the icon of the message
+ * @param translationParameters Optional array parameter.
+ *                              Used as parameters for the translation of translationKey.
+ *                              When this parameters is absent, the user sending the message is used as translation
+ *                              parameter.
  * @since 10.5RC1
  * @since 9.11.6
  *#
-#macro(displayMessageNotification $compositeEvent $translationKey $icon)
+#macro(displayMessageNotification $compositeEvent $translationKey $icon $translationParameters)
```

Comment:
I'm not sure how we handle this, since velocity allow to call a macro with a missing parameter, but maybe it would be better for tracability to have a new macro with the new parameter and a proper since on it?

Reply:
Replaced by `displayGroupMessageNotification` a dedicated macro for the messages sent to a group.